src/routes/AuthRoutes.tsx

```
@@ -32,7 +32,12 @@ const routes = [
 		path: '/document-scanner',
 		component: () => {
 			const { userData } = useContext(AuthContext)!;
-			return <DocumentScanner userId={userData?.user_id} userData={userData?.docs} />;
+			return (
+				<DocumentScanner
+					userId={userData?.user_id}
+					userData={userData?.docs}
+				/>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)